src/styles/typography.scss

```diff
@@ -1,5 +1,6 @@
 @import "./theme-mixin";
-$h1-legacy-font-size: 32px;
+@import "~monday-ui-style/dist/mixins";
```

Comment:
@SergeyRoyt - is it possible to create ab import path without the `dist` folder?

Reply:
tried to add it to the exports and it did not work for me. If you have any idea I would love to hear @SergeyRoyt @shlomitc

Reply:
Perhaps we could just add separate endpoint in `monday-ui-style` `package.json` `exports`, no?

Reply:
Did it and did not work, maybe related to sass loaders?